cmd/createFgaModel.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright © 2024 NAME HERE <EMAIL ADDRESS>
+*/
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+
+	authz "github.com/canonical/identity-platform-login-ui/internal/authorization"
+	logging "github.com/canonical/identity-platform-login-ui/internal/logging"
+	monitoring "github.com/canonical/identity-platform-login-ui/internal/monitoring"
+	fga "github.com/canonical/identity-platform-login-ui/internal/openfga"
+	tracing "github.com/canonical/identity-platform-login-ui/internal/tracing"
+	"github.com/spf13/cobra"
+)
+
+// createFgaModelCmd represents the createFgaModel command
+var createFgaModelCmd = &cobra.Command{
+	Use:   "create-fga-model",
+	Short: "Creates an openfga model",
+	Long:  `Creates an openfga model`,
+	Run: func(cmd *cobra.Command, args []string) {
+		apiUrl, _ := cmd.Flags().GetString("api-url")
+		apiToken, _ := cmd.Flags().GetString("api-token")
+		storeId, _ := cmd.Flags().GetString("store-id")
+		createModel(apiUrl, apiToken, storeId)
+	},
+}
+
+func init() {
+	rootCmd.AddCommand(createFgaModelCmd)
+
+	createFgaModelCmd.PersistentFlags().String("api-url", "", "The openfga API URL")
+	createFgaModelCmd.PersistentFlags().String("api-token", "", "The openfga API token")
+	createFgaModelCmd.PersistentFlags().String("store-id", "", "The openfga store to create the model in")
+	createFgaModelCmd.MarkPersistentFlagRequired("api-url")
+	createFgaModelCmd.MarkPersistentFlagRequired("api-token")
+	createFgaModelCmd.MarkPersistentFlagRequired("store-id")
+}
+
+func createModel(apiUrl, apiToken, storeId string) {
+	logger := logging.NewNoopLogger()
```

Comment:
The reason that i didn't use the logger is that we don't want any of the features the logger provides here. The logger is configured to write to a file and add a timestamp along with other sugar-coating. IMO for a CLI command all we want is to print to the CLI, so even if we didn't forward the logs to a file it would not be needed. 

I thought about adding a `plain` logger struct, but I decided against it because I believe that these messages aren't exactly "logs". They are a command's output. Not sure if makes sense